StudentManagement/StudentManagement.Console/Program.cs

```diff
@@ -61,8 +90,23 @@
 
             }
 
+
+            break;
+        case 3:
+            if (!File.Exists(filePath))
+            {
+                Console.WriteLine("Creating a new json file...");
+                File.Create(filePath).Close();
+
+            }
             break;
+        case 4:
 
+            File.WriteAllText(filePath, studentJson, Encoding.UTF8);
```

Comment:
you create studentJson file at the beginning of the do/while loop. It would be better to move serializing data here instead.

Reply:
Yes i did. code looks like that now:
case 3:
            if (!File.Exists(filePath))
            {
                Console.WriteLine("Creating a new json file");
                File.Create(filePath).Close();
            }
            var studentJson = JsonConvert.SerializeObject(students);
            File.WriteAllText(filePath, studentJson, Encoding.UTF8);